corporate/views/remote_billing_page.py

```diff
@@ -206,15 +206,15 @@ def remote_realm_billing_finalize_login(
         raise
     except Exception:  # nocoverage
         billing_logger.exception(
-            "%s: Failed to migrate customer from server (id: %s) to realms",
+            "%s: Failed to migrate customer from server (id: %s) to realm",
             request.path,
             remote_server.id,
             stack_info=True,
         )
         raise JsonableError(
             _(
-                "Failed to migrate customer from server to realms. Please contact support for assistance."
-            )
+                "Couldn't reconcile billing data between server and realm. Please contact {support_email}"
```

Comment:
This is the same error string that is used in `handle_customer_migration_from_server_to_realm` when a plan on the remote server can't be migrated to the single, human realm. Thought it made sense to reuse that here since these strings are marked for translation.